day-3/index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Inspiration</title>
+</head>
+<body>
+  <script src="build/react.js"></script>
+  <script src="build/react-dom.js"></script>
```

Comment:
Where is the build directory coming from? Include a package.json in your PR next time that brings in React and react-dom.